src/Response.php

```diff
@@ -74,6 +74,6 @@ public function toResponse($request)
             ]);
         }
 
-        return View::make($this->rootView, $this->viewData + ['page' => $page]);
+        return ResponseFacade::view($this->rootView, $this->viewData + ['page' => $page]);
```

Comment:
Having to do this makes me think we should consider renaming `Inertia\Response` to `Inertia\InertiaResponse`.

What do you think @reinink?

Reply:
Didn't we rename it before from `InertiaResponse` to `Response` so it wouldn't  be verbose anymore? 🤔

Reply:
Yeah I'm not really a fan. Isn't that the whole point of namespaces? I get it though, it feels weird in this situation to have a `ResponseFacade`. 

I think the better naming here is rename `ResponseFacade` to `ResponseFactory`, because that's actually what is is.

```php
use Illuminate\Support\Facades\Response as ResponseFactory
```
```php
return ResponseFactory::view($this->rootView, $this->viewData + ['page' => $page]);
```

Thoughts?

Reply:
Wouldn't it get confused with Inertia's own `Inertia\ResposeFactory`?

Reply:
Renamed the Façade alias to `ResponseFactory` as suggested by @reinink

Reply:
Sounds good 👍